hibernate-core/src/test/java/org/hibernate/test/loadplans/walking/LoggingAssociationVisitationStrategy.java

```diff
@@ -23,17 +29,32 @@
 /**
  * @author Steve Ebersole
  */
-public class LoggingAssociationVisitationStrategy implements AssociationVisitationStrategy {
+public class LoggingAssociationVisitationStrategy extends FetchStyleLoadPlanBuildingAssociationVisitationStrategy {
 	private int depth = 1;
 
+	/**
+	 * Constructs a FetchStyleLoadPlanBuildingAssociationVisitationStrategy.
```

Comment:
seems we are constructing a `LoggingAssociationVisitationStrategy` here, so the above Javadoc is a little bit confusing